tee-worker/app-libs/stf/src/trusted_call_result.rs

```diff
@@ -101,3 +101,12 @@ pub struct RequestVCResult {
 	pub pre_mutated_id_graph: AesOutput,
 	pub pre_id_graph_hash: H256,
 }
+
+#[derive(Debug, Encode, Decode, Clone)]
+pub struct RequestVcResultOrError {
+	pub payload: Vec<u8>,
+	pub is_error: bool,
+	// When `is_error` true, the following `idx` and `len` should be ignored.
+	pub idx: u8,
+	pub len: u8,
```

Comment:
If this is the case, How should clients map the errors back to the corresponding assertion request?

Reply:
Besides, when response is error, and both idx and len are 0: it means error happens before any assertion processing.